### PR TITLE
added color to triggers

### PR DESCRIPTION
**Pull request by davidp57**
Hi,

I added the ability to set a color for a trigger.

See this in action here:
image

### CODE REVIEW

File: dcs/triggers.py [hunk 1]
```diff
@@ -11,6 +11,11 @@
 if TYPE_CHECKING:
     from dcs.terrain import Terrain
 
+class TriggerColor(Enum):
+    Red = "0xff0000ff"
```
Comment (rp-) [inline dcs/triggers.py:15]: Maybe add the `White` also here?
I guess the rest of the colors are the same as in DCS predefined?

File: dcs/triggers.py [hunk 2]
```diff
@@ -197,6 +202,7 @@ class TriggerRule:
 
     def __init__(self, event: Event, comment: str):
         self.comment: str = comment
+        self.color: str = "0x00000000"
```
Comment (rp-) [inline dcs/triggers.py]: and then also use the `TriggerColor.White` here

**Review by rp- — CHANGES_REQUESTED** [review]

File: dcs/triggers.py (hunk 2 above)
Comment (davidp57) [reply dcs/triggers.py]: Hello,
This has been done in 18eb279!
Thanks.

Comment (rp-) [reply dcs/triggers.py]: Seems I looked at the wrong diff, sorry

**Comment by rp-** [comment]
Just fix the lint errors and we are good to go

**Comment by davidp57** [comment]
done (I think)

**Pull request merged**